src/lib/routes/admin-api/project/variants.ts

```diff
@@ -41,7 +41,6 @@ export default class VariantsController extends Controller {
         this.route({
             method: 'get',
             path: PREFIX,
-            acceptAnyContentType: true,
```

Comment:
Still need `acceptAnyContentType` for `get`, should really improve on that.

Reply:
Oh, ok. thanks